apps/portal/app/routes/readonly+/identity+/$id+/index.tsx

```diff
@@ -1,64 +1,211 @@
-    positions: getPositionsOnIdentity({
-      request,
-      identityId: id,
-      searchParams,
-    }),
-    claims: getClaimsAboutIdentity({
-      request,
-      identityId: id,
-      searchParams,
-    }),
-    claimsSummary: fetchWrapper(request, {
-      method: ClaimsService.claimSummary,
-      args: {
-        identity: id,
+  const queryClient = new QueryClient()
+
+  const triplesLimit = parseInt(url.searchParams.get('claimsLimit') || '10')
+  const triplesOffset = parseInt(url.searchParams.get('claimsOffset') || '0')
+  const triplesOrderBy = url.searchParams.get('claimsSortBy')
+
+  const triplesWhere = {
+    _or: [
+      {
+        subjectId: {
+          _eq: id,
+        },
+      },
+      {
+        objectId: {
+          _eq: id,
+        },
       },
-    }),
+      {
+        predicateId: {
+          _eq: id,
+        },
+      },
+    ],
+  }
+
+  const positionsLimit = parseInt(
+    url.searchParams.get('positionsLimit') || '10',
+  )
+  const positionsOffset = parseInt(
+    url.searchParams.get('positionsOffset') || '0',
+  )
+  const positionsOrderBy = url.searchParams.get('positionsSortBy')
+
+  const positionsWhere = {
+    vaultId: { _eq: id },
+  }
+
+  await queryClient.prefetchQuery({
+    queryKey: ['get-atom', { id: params.id }],
+    queryFn: () =>
+      fetcher<GetAtomQuery, GetAtomQueryVariables>(GetAtomDocument, {
+        id: params.id,
+      })(),
+  })
+
+  await queryClient.prefetchQuery({
+    queryKey: [
+      'get-triples-with-positions',
+      { triplesWhere, triplesLimit, triplesOffset, triplesOrderBy },
+    ],
+    queryFn: () =>
+      fetcher<
+        GetTriplesWithPositionsQuery,
+        GetTriplesWithPositionsQueryVariables
+      >(GetTriplesWithPositionsDocument, {
+        where: triplesWhere,
+        limit: triplesLimit,
+        offset: triplesOffset,
+        orderBy: triplesOrderBy ? [{ [triplesOrderBy]: 'desc' }] : undefined,
+      })(),
+  })
+
+  await queryClient.prefetchQuery({
+    queryKey: [
+      'get-atom-positions',
+      { positionsWhere, positionsLimit, positionsOffset, positionsOrderBy },
+    ],
+    queryFn: () =>
+      fetcher<GetPositionsQuery, GetPositionsQueryVariables>(
+        GetPositionsDocument,
+        {
+          where: positionsWhere,
+          limit: positionsLimit,
+          offset: positionsOffset,
+          orderBy: positionsOrderBy
+            ? [{ [positionsOrderBy]: 'desc' }]
+            : undefined,
+        },
+      )(),
+  })
+
+  return json({
+    initialParams: {
+      triplesLimit,
+      triplesOffset,
+      triplesOrderBy,
+      triplesWhere,
+      positionsLimit,
+      positionsOffset,
+      positionsOrderBy,
+      positionsWhere,
+      atomId: params.id,
+    },
   })
 }
 
 export default function ReadOnlyProfileDataAbout() {
-  const { positions, claims, claimsSummary } = useLiveLoader<typeof loader>([
-    'attest',
-  ])
+  const { initialParams } = useLiveLoader<typeof loader>(['attest'])
```

Comment:
I meant to ask you this yesterday -- do you think we still need the `useLiveLoader` instead of `useLoaderData` at this point? I've been leaving them in, but wondering if maybe we do a pass to convert these all at once when the invalidation patterns are in place?

Reply:
I'd say we drop LiveLoader since we are using React Query now for invalidation.